docs/user-guide/annotations.md

```diff
@@ -26,3 +26,12 @@ resource "aws_instance" "foo" {
     instance_type = "t1.2xlarge"
 }
 ```
+
+It is possible to add a reason for why a rule is ignored:
+
+```hcl
+resource "aws_instance" "foo" {
+    # tflint-ignore: aws_instance_invalid_type # A good reason
+    instance_type = "t1.2xlarge"
+}
+```
```

Comment:
It would be nice to mention what types of comment formats are allowed.
In my understanding, you can add comments after the `#` and `//` at the end of the annotation comment.

Strictly speaking, comments starting with `*` or `/` are probably valid for reasons of regexp, but I would like to make it clear that this is not what we intended.